### PR TITLE
Change property names

### PR DESCRIPTION
**Pull request by ukenhk-kuni077**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -95,13 +95,11 @@ export default {
     .then(response=>response.json())
     .then(json=>{
       console.log(JSON.stringify(json,null,'\t'));
-      this.taskDatas = json.map(x=>Object.assign(x,{
-        title : x.todoText,
-        regDate:x.regDate.replace(/\//g,'-')
```
Comment (amanoese) [inline src/App.vue]: regDate format is different of html5 specification.
need to data convert on either db or js.
this code is need.

**Review by amanoese — CHANGES_REQUESTED** [review]
thank you for pull request.
but you need a little this code fix.

**Comment by ukenhk-kuni077** [comment]
pull request の方のコメントに書かなかったかもしれませんが、日付は db 側の値を直しました。
( ハイフン区切りに修正した )

**Comment by amanoese** [comment]
Conflict happened occurred by this pull request.
Please merge my master branch into the pull request branch.

The cause is your master branch and my master branch are different.
Your master branch was not merge my master branch using fast-forward few days ago.

**Comment by ukenhk-kuni077** [comment]
コンフリクト解消しました。

**Pull request merged**